peekingduck/pipeline/nodes/input/visual.py

```diff
@@ -242,7 +242,9 @@ def _get_next_frame(self) -> Dict[str, Any]:
             "img": None,
             "filename": self._file_name if self._file_name else self.filename,
             "pipeline_end": True,
-            "saved_video_fps": self._fps if self._fps > 0 else self.saved_video_fps,
+            "saved_video_fps": self._fps
+            if (self._fps > 0 and self._fps <= 200)
```

Comment:
Can recode `(self._fps > 0 and self._fps <= 200)` in the pythonic way `(0 < self._fps <= 200)`.

Reply:
fixed